### ISSUE TITLE
Add demo application parallel to lib

### ISSUE THREAD

**Issue by shlomiassaf** [preboot/angular2-library-seed#9]
While developing a library it is always helpful to use an application that consumes that library within the same project.

The most obvious reasons are:
- Speeds up developments in a large factor
- Easy debugging
- Provides examples to end user
- A **github.io** site for presentation (script to upload can be easily integrated)

Suggested Structure:

```
src
---lib
---demo-app
```

Webpack is used for development and for bundling the output for the `github.io` site.
Rollup is used for NPM package creation.

**Comment by NathanWalker** [comment]
demo app exists here: https://github.com/preboot/angular2-library-seed/tree/master/src
But I assume you are going for a completely separated demo app which would consume the npm lib and module.

**Comment by shlomiassaf** [comment]
Yep, see my repo https://github.com/shlomiassaf/angular2-modal/tree/master/src , disregard naming...

**Comment by NathanWalker** [comment]
@shlomiassaf a PR sometime would be appreciated; the seed setup at the moment just mirrors https://github.com/preboot/angular2-webpack right now for ease of use and updating.
